src/components/html/EcAdminSettingsForm.html

```diff
@@ -87,14 +87,13 @@
             }) in parseAdminSettingsField(config)"
             v-if="component"
             :key="name"
-            :label="schema.title"
+            :label="checkComplexSchema(schema)? '' : schema.title"
```

Comment:
Mas aí sempre que for um nested object vai ficar sem título/descrição aqui?
Nem sempre vai ter um input group específico pro objeto, chegou a testar em outros casos?